instance/models/mixins/secret_keys.py

```diff
@@ -114,12 +141,47 @@ def secret_key(self):
         """
         return b64decode(self.secret_key_b64encoded)
 
+    @property
+    def rsa_key(self):
+        """
+        Return the RSA key as a Cryptodome.RSA object.
+        """
+        if not self.secret_key_rsa_private:
```

Comment:
@giovannicimolin, doesn't it make sense to generate it if it doesn't exist?

Reply:
Yes! I'll make those changes.